src/components/side_panel/side_panel/side_panel.xml

```diff
@@ -3,21 +3,21 @@
     <div class="o-sidePanel">
       <div class="o-sidePanelHeader">
         <div class="o-sidePanelTitle" t-esc="getTitle()"/>
-        <div class="o-sidePanelClose" t-on-click="() => this.props.onCloseSidePanel()">✕</div>
+        <div class="o-sidePanelClose" t-on-click="close">✕</div>
```

Comment:
```suggestion
        <div class="o-sidePanelClose" t-on-click="sidePanelStore.close">✕</div>
```
nitpick

Reply:
it would need to be binded `sidePanelStore.close.bind(sidePanelStore)` :/